metrics/prometheus_test.go

```diff
@@ -55,7 +55,11 @@ func (p testSubcontainersInfoProvider) SubcontainersInfo(string, *info.Container
 				Aliases: []string{"testcontaineralias"},
 			},
 			Spec: info.ContainerSpec{
-				Image:        "test",
+				Image: "test",
```

Comment:
You need to add in `HasCpu: true,` in here so that the CPU specs are output as metrics. You'll probably find then that there are a couple of other metrics which haven't been tested for that you'll need to insert into `metrics/testdata/prometheus_metrics` to get this to pass.